frontends/precinct-scanner/src/screens/poll_worker_screen.test.tsx

```diff
@@ -126,7 +139,7 @@ describe('shows Export Results button only when polls are closed and more than 0
 
 describe('shows Livecheck button only when enabled', () => {
   test('enable livecheck', async () => {
-    mocked(isLiveCheckEnabled).mockReturnValue(true);
+    mocked(isFeatureFlagEnabled).mockReturnValue(true);
```

Comment:
This testing approach seems like it could be a bit unpredictable. E.g. if another feature flag is added that influences this component, this mock will secretly change its behavior.

Maybe we can have a test helper to mock feature flags. E.g.:

```ts
const featureFlags = mockFeatureFlags()

beforeEach(() => featureFlags.reset())

test(..., () => {
  featureFlags.enable(featureFlagName)
```

Reply:
Yes I was thinking we should have some kind of test helper but couldn't quite figure out how it should work.

Reply:
https://github.com/votingworks/vxsuite/issues/2548